src/test.zig

```diff
@@ -203,6 +203,11 @@ test "Node" {
     try testing.expectEqual(0, node.startPoint().column);
     try testing.expectEqual(13, node.endPoint().column);
 
+    try testing.expectEqual(node.startByte(), node.range().start_byte);
+    try testing.expectEqual(node.endByte(), node.range().end_byte);
+    try testing.expectEqual(node.startPoint(), node.range().start_point);
+    try testing.expectEqual(node.endPoint(), node.range().end_point);
```

Comment:
```suggestion
    var range = node.range();
    try testing.expectEqual(0, range.start_byte);
    try testing.expectEqual(13, range.end_byte);
    try testing.expectEqual(0, range.start_point);
    try testing.expectEqual(13, range.end_point);
```

Reply:
@ObserverOfTime thanks for the review, I updated in the test in dece4e50a2d20726aaab2c0e38eba97cb97623f6.